dvc/remote/base.py

```diff
@@ -275,8 +275,10 @@ def load_dir_cache(self, checksum):
             raise DirCacheError(checksum) from exc
 
         if not isinstance(d, list):
-            msg = "dir cache file format error '{}' [skipping the file]"
-            logger.error(msg.format(relpath(path_info)))
+            logger.error(
+                "dir cache file format error '{}' [skipping the file]",
+                path_info,
+            )
```

Comment:
Still use msg maybe?
```python
msg = "dir cache file format error '{}' [skipping the file]"
logger.error(msg, path_info)
```

Reply:
No point in a new var, it is an old code from time when we weren't sure on formatting. We are inlining it these days, so it makes sense to inline here as well.